hosts/unittester.py

```diff
@@ -4,11 +4,17 @@
 
 import unittest
 
-from juno.junocam   import Test_Juno_Junocam
-#from cassini.iss    import Test_Cassini_ISS
-#from hst            import Test_HST
+from hosts.juno.unittester            import *
+#from hosts.cassini.unittester         import *
+#from hosts.hst.unittester             import *
+#from hosts.voyager.unittester         import *
+#from hosts.newhorizons.unittester     import *
+#from hosts.keck.unittester            import *
```

Comment:
Do we care what flake8 thinks of the unit tests?  In this case, * seems to make sense because we actually intend to run every test contained in the module.  Without *, we have to explicitly add any new tests to this file when we add them to the referenced module.  I don't see the sense in that.

Reply:
I agree. I think this is one of the rare cases where import * makes perfect sense.